vm/runtime.ts

```diff
@@ -421,7 +421,7 @@ module J2ME {
     ctx: Context;
     allCtxs: Set<Context>;
 
-    isolate: com.sun.cldc.isolate.Isolate;
+    isolate: any;
```

Comment:
It is defined at https://github.com/mozilla/j2me.js/blob/master/bindings.ts.in#L428